docs/source/serving/openai_compatible_server.md

```diff
@@ -189,8 +189,7 @@ The order of priorities is `command line > config file values > defaults`.
 ---
 
 ## Tool calling in the chat completion API
-
-vLLM supports named function calling and `auto` tool choice  in the chat completion API. The `tool_choice` options `required` is **not yet supported** but on the roadmap.
+vLLM supports only named function calling in the chat completion API. The `tool_choice` option `required` is **not yet supported** but on the roadmap.
```

Comment:
Is it intentional to remove the mention of `auto` tool choice? This change gives me the impression that it's not supported.

Reply:
Yes it is intentional. The original sentence was referring to the `auto` and `required` **NOT** being supported, while the `auto` tool choice option is actually supported.

Reply:
Maybe a better change would be to be more explicit and actually mention that the `none` and `auto` tool_choices options are supported while the `required` option is not yet supported and in the roadmap ?

Reply:
ha yes, I see what you mean, because I haven't also removed the `only` it doesn't give a very clear picture of what actually vLLM supports or not. I'll update

Reply:
Updated !